aiven_mysql_migrate/exceptions.py

```diff
@@ -15,6 +15,10 @@ class WrongMigrationConfigurationException(MigrationPreCheckException):
     pass
 
 
+class UnsupportedMigrationParameterException(WrongMigrationConfigurationException):
```

Comment:
Maybe we don't need another exception for this? I would use the same `WrongMigrationConfigurationException`
The meaning is pretty much the same - wrong configuration, the message from exception will show what exactly is wrong. Also in this case we won't need another mapping to error code where we actually use aiven-mysql-migrate and no need to communicate new error code to the stakeholders.
I don't think semantic is different for case when port is invalid or parameter is invalid.